Duplicati/Library/Backend/Dropbox/DropboxHelper.cs

```diff
@@ -336,16 +339,19 @@ public class MetaData
         [JsonProperty(".tag")]
         public string tag { get; set; }
         public string name { get; set; }
-        public string path_lower { get; set; }
-        public string path_display { get; set; }
-        public string id { get; set; }
-
         public string server_modified { get; set; }
-        public string rev { get; set; }
         public ulong size { get; set; }
-
         public bool IsFile { get { return tag == "file"; } }
 
+        // While this is unused, the Dropbox API v2 documentation does not
+        // declare this to be optional.
+        // ReSharper disable once UnusedMember.Global
+        public string id { get; set; }
+
+        // While this is unused, the Dropbox API v2 documentation does not
+        // declare this to be optional.
+        // ReSharper disable once UnusedMember.Global
+        public string rev { get; set; }

```

Comment:
While everything still appears to work when the `id` and `rev` fields are removed, I left them in because the documentation does not claim that they are optional (in contrast to the fields I removed above):

https://www.dropbox.com/developers/documentation/http/documentation#files-upload